src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaClientFetchFactory.java

```diff
@@ -978,8 +983,9 @@ else if (length != 0)
             {
                 if (KafkaConfiguration.DEBUG)
                 {
-                    System.out.format("[client] [0x%016x] %s[%d] FETCH RecordBatch %d\n",
-                            client.replyId, client.topic, client.partitionId, recordBatch.length());
+                    System.out.format("[client] [0x%016x] %s[%d] FETCH RecordBatch %d %d %d %04x\n",
+                            client.replyId, client.topic, client.partitionId, recordBatch.baseOffset(),
+                        recordBatch.lastOffsetDelta(), recordBatch.length(), recordBatch.attributes());
```

Comment:
Please remove the attributes from the debug output, we are processing them correctly for control batch and compressed batch.